routes/teamAPI.js

```diff
@@ -46,17 +53,20 @@ router.get('/check',teamController.checkExist);
 //	optional:
 //		name:String
 //		description:String
-//		coach:String (user)
-//		lead:String(user)
-//		member:[String] (user)
-//		project: [String] (project)
+//		coach:id (user)
+//		lead:id(user)
+//		member:[id] (user)
+//		project: id (project)
 //response:
 //{team:TeamEntity}
 router.post('/update/:id',authenticator.authenticate,teamController.update);
 
-router.get('/member/:id', teamController.getByUserAsMember);
-router.get('/coach/:id', teamController.getByUserAsCoach);
-router.get('/lead/:id', teamController.getByUserAsLead);
+//body parameters
+//	required:
+//		id:id(team)
+//response:
+//{success:true}
+router.delete('/remove/:id',authenticator.authenticate,teamController.removeById);
```

Comment:
why do you want to use "delete" method here?
Since you have 'remove' word in your path, a "get" method is ok either.